clang/test/CXX/drs/cwg27xx.cpp

```diff
@@ -18,6 +30,38 @@ void f(B b) {
 struct D : B {};
 } // namespace cwg2718
 
+namespace cwg2749 { // cwg2749: 19
+
+extern int x[2];
+struct Y {
+  int i;
+  int j;
+};
+extern Y y[2];
+
+#if __cplusplus >= 201103L
```

Comment:
If you need to match diagnostics only in C++98, you should modify C++98 RUN line to say `-verify=expected,cxx98`, and then use directives like `// cxx98-error`.